Gemfile

```diff
@@ -1,4 +1,6 @@
 source 'https://rubygems.org'
 
+gem 'json-streamer', :git => "git://github.com/thisismydesign/json-streamer"
```

Comment:
https://github.com/abonas/kubeclient/pull/254/commits/6d46d6d82cd709f975cff97279a6a8bfeca94060 I saw it was removed in another commit (which also looks redundant)